src/components/savedLocations/components/SavedLocations.js

```diff
@@ -0,0 +1,126 @@
+        console.log('handling it');
+    };
+    const renderPathUrl = savedLocationId => `/dashboard/savedLocations/${savedLocationId}`;
+    const renderList = () => (
+        <List
+            itemLayout="vertical"
+            size="small"
+            pagination={{
+                onChange: page => {
+                    console.log(page);
+                },
+                pageSize: 3,
+            }}
+            dataSource={listData}
+            renderItem={item => (
+                <List.Item
+                    key={item.title}
+                    actions={[
+                        <Tooltip placement="top" title="Edit Details" key="list-vertical-edit-o">
+                            <Button
+                                onClick={() => editSavedLocation(item.savedLocationId)}
+                                type="link"
+                                icon="edit-o"
+                            />
+                        </Tooltip>,
+                        <Tooltip placement="top" title="View details" key="list-vertical-eye-o">
+                            <Button
+                                onClick={() => viewSavedLocation(item.savedLocationId)}
+                                type="link"
+                                icon="eye-o"
+                            />
+                        </Tooltip>,
+                        <Tooltip placement="top" title="Delete" key="list-vertical-delete-o">
+                            <Button
+                                onClick={() => deleteSavedLocation(item.savedLocationId)}
+                                type="link"
+                                icon="delete-o"
+                            />
+                        </Tooltip>,
+                    ]}
+                    extra={(
+                        <div className="d-flex justify-content-between">
+                            <Typography.Text type="secondary">
+                                {LOCATION_NUMBER_LABEL}
+                            </Typography.Text>
+                            <Tag color="volcano" className="ml-3">
+                                {item.locations.length}
+                            </Tag>
+                        </div>
+                    )}
+                >
+                    <List.Item.Meta
+                        title={(
+                            <Link to={renderPathUrl(item.savedLocationId)}>
+                                {item.title}
+                            </Link>
+                        )}
+                        description={item.description}
+                    />
+                </List.Item>
+            )}
+        />
+    );
+    for (let i = 0; i < 9; i++) {
+        listData.push({
+            description:
+                'Ant Design, a design language for background ',
+            locations: [1, 2, 3],
+            savedLocationId: 'jjajsnksashsd',
+            title: `ant design part ${i}`,
+        });
+    }
```

Comment:
if this is dummy data, it should be in constants

Reply:
Alright